server/middleware/setUpWebSession.ts

```diff
@@ -10,7 +10,7 @@ import logger from '../../logger'
 const RedisStore = connectRedis(session)
 
 export default function setUpWebSession(): Router {
-  const client = createRedisClient(`middleware/setUpWebSession.ts`, true)
```

Comment:
but does this client expect legacy mode? did sessions work for you locally?

Reply:
Works fine locally, unit tests pass and integration tests also pass.

I'm having a quick look at `legacyMode` and AFAICT it would cause problem if trying to talk with Redis before it's connected. 

- It seems like in `setUpWebSession()` (only place passing `legacyMode: true`) [it calls `connect()` explicitly](https://github.com/ministryofjustice/hmpps-incentives-ui/blob/2d61da9798e381f9c580b2c18d6e45703988615c/server/middleware/setUpWebSession.ts#L13-L14)
- `HmppsAuthClient`/`TokenStore` seem to also [call connect before talking to the client, in `ensureConnected()`](https://github.com/ministryofjustice/hmpps-incentives-ui/blob/2d61da9798e381f9c580b2c18d6e45703988615c/server/data/tokenStore.ts#L8-L11)

Everything is asynchronous tho and timing could mean the fact is running fine locally is not worth much?